IC_test.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import os
+from numpy.random import randint
+from IC import IndustrialConsumer
+
+## Market Conditions
+
+demand = np.zeros(48)
+for i in range(48):
+    demand[i] = 10
+    #demand[i] = randint(1,51)
+
+cost_elec_market = np.zeros(48)
+for i in range(48):
+    cost_elec_market[i] = 0
+    #cost_elec_market[i] = randint(30,51)
+
+## The Game
+
+IC = IndustrialConsumer(demand,cost_elec_market)

```

Comment:
you should not use capital letters for instances of a class, besides, I suggest explicit names:
```suggestion
industrial_consumer = IndustrialConsumer(demand,cost_elec_market)

```